src/clusterfuzz/_internal/system/environment.py

```diff
@@ -1070,6 +1070,12 @@ def is_local_development():
   return bool(get_value('LOCAL_DEVELOPMENT') and not get_value('PY_UNITTESTS'))
 
 
+def is_production():
+  """Return True if running in production environment (e.g. not running
+  a bot locally, excludes tests)."""
```

Comment:
I think this comment is a bit confusing. During testing, `is_production` returns True? Can you be more explicit about this?

Reply:
Agreed it was not representative of the function's logic.

Done.

Reply:
You may want to double-check it though.